### ISSUE TITLE
Auto-sync on GUI startup not working

### ISSUE THREAD

**Issue by FongTsang** [bpozdena/OneDriveGUI#61]
Dear developer, I am using onedriveGUI on archlinux+GNOME, everything works fine except for Auto-sync on GUI startup, every time I re-launch that application, the checkbox of Auto-sync on GUI startup not working restore status of uncheck. is there any method to fix it? thanks a lot

**Comment by bpozdena** [comment]
Please provide the bellow details:
1) Output of `cat ~/.config/onedrive-gui/profiles`
2) Does it work when you manually edit the config file and set `auto_sync = True` ?
3) Screenshot of the affected GUI option
4) OneDrive GUI log file collected after app startup. The log should be in `/tmp/onedrive-gui/onedrive-gui.log`
5) How are you launching the GUI? From AppImage?

**Comment by FongTsang** [comment]
> Please provide the bellow details:
> 
>     1. Output of `cat ~/.config/onedrive-gui/profiles`
> 
>     2. Does it work when you manually edit the config file and set `auto_sync = True` ?
> 
>     3. Screenshot of the affected GUI option
> 
>     4. OneDrive GUI log file collected after app startup. The log should be in `/tmp/onedrive-gui/onedrive-gui.log`
> 
>     5. How are you launching the GUI? From AppImage?

Thanks a lot, problem solved by manually edit `auto_sync = True` in `~/.config/condrive-gui/profiles`, by the way, I install it via 
`yaourt`, there is the access permission of the configure:
![图片](https://user-images.githubusercontent.com/53118840/197776805-ed99af69-db81-4955-a10b-adb64c55868f.png)
Does the problem has something to do with the access permission?

**Comment by bpozdena** [comment]
Great. Thanks for confirming.

The file permissions are correct. Also the fact that the files exist suggest the GUI can read/write to them. I just tested the GUI installed from AUR by using Pacman on Manjaro Gnome , and I did not encounter any issues with the auto-sync feature. 

Perhaps consider switching away from yaourt as it is no longer listed as [AUR helper](https://wiki.archlinux.org/title/AUR_helpers). Or try using the OneDriveGUI AppImage. Unless there is some specific error in the GUI logs, I'm afraid there isn't much more I can do.
